modules/apps/transfer/types/token.go

```diff
@@ -25,3 +26,13 @@ func (t Token) Validate() error {
 
 	return nil
 }
+
+func (t Token) ConvertToCoin() (sdk.Coin, error) {
```

Comment:
Needs godoc. 

I think if we're going to add this functionality we should go with `ToCoin` naming

Reply:
Resolve command. 
Thanks